package.json

```
@@ -9,14 +9,17 @@
     "lint": "next lint",
     "test": "jest tests/unit",
     "lint:fix": "eslint --fix */**/*.js tests/**/*.js",
-    "cy:open": "cypress open --env RELEASE_VERSION=full --config baseUrl=http://localhost:3000",
-    "cy:open:mvp": "cypress open --env RELEASE_VERSION=mvp --config baseUrl=http://localhost:3000,testFiles=**/*Mvp.spec.js",
-    "cy:run": "cypress run --env RELEASE_VERSION=full --config baseUrl=http://localhost:3000",
-    "cy:run:mvp": "cypress run --env RELEASE_VERSION=mvp --config baseUrl=http://localhost:3000 --spec '**/*Mvp.spec.js'",
+    "cy:open": "cypress open --env RELEASE_VERSION=full,ENABLE_LEASEHOLDER_FLOW=false --config baseUrl=http://localhost:3000",
+    "cy:open:enableLeaseholderFlow": "cypress open --env RELEASE_VERSION=full,ENABLE_LEASEHOLDER_FLOW=true --config baseUrl=http://localhost:3000",
+    "cy:open:mvp": "cypress open --env RELEASE_VERSION=mvp,ENABLE_LEASEHOLDER_FLOW=false --config baseUrl=http://localhost:3000,testFiles=**/*Mvp.spec.js",
+    "cy:run": "cypress run --env RELEASE_VERSION=full,ENABLE_LEASEHOLDER_FLOW=false --config baseUrl=http://localhost:3000",
+    "cy:run:mvp": "cypress run --env RELEASE_VERSION=mvp,ENABLE_LEASEHOLDER_FLOW=false --config baseUrl=http://localhost:3000 --spec '**/*Mvp.spec.js'",
+    "cy:run:enableLeaseholderFlow": "cypress run --env RELEASE_VERSION=full,ENABLE_LEASEHOLDER_FLOW=true --config baseUrl=http://localhost:3000",
     "test:integration": "start-server-and-test dev http://localhost:3000 cy:open",
-    "test:integration:mvp": "env RELEASE_VERSION=mvp start-server-and-test dev http://localhost:3000 cy:open:mvp",
+    "test:integration:mvp": "env RELEASE_VERSION=mvp,ENABLE_LEASEHOLDER_FLOW=false start-server-and-test dev http://localhost:3000 cy:open:mvp",
+    "test:integration:enableLeaseholderFlow" : "env RELEASE_VERSION=full,ENABLE_LEASEHOLDER_FLOW=true start-server-and-test dev http://localhost:3000 cy:open:enableLeaseholderFlow",
     "test:headless:integration": "start-server-and-test dev http://localhost:3000 cy:run",
-    "test:headless:integration:mvp": "env RELEASE_VERSION=mvp start-server-and-test dev http://localhost:3000 cy:run:mvp"
+    "test:headless:integration:mvp": "env RELEASE_VERSION=mvp,enable_LEASEHOLDER_FLOW=false start-server-and-test dev http://localhost:3000 cy:run:mvp"
```

The additional env var should have a consistent case (to itself and others added above too).
```suggestion
    "test:headless:integration:mvp": "env RELEASE_VERSION=mvp,ENABLE_LEASEHOLDER_FLOW=false start-server-and-test dev http://localhost:3000 cy:run:mvp"
```